apm-sniffer/apm-sdk-plugin/pom.xml

```diff
@@ -202,4 +200,31 @@
             </plugin>
         </plugins>
     </build>
+
+    <profiles>
+        <profile>
+            <!-- ojdbc is not Apache license compatible, so it must be manually downloaded first -->
+            <id>include-ojdbc</id>
+            <activation>
+                <file>
+                    <exists>${basedir}/../../ci-dependencies/ojdbc14-10.2.0.4.0.jar</exists>
+                </file>
+            </activation>
+            <modules>
+                <module>oracle-10.x-plugin</module>
+            </modules>
+        </profile>
+        <profile>
+            <!-- resin is not Apache license compatible, so it must be manually downloaded first -->
+            <id>include-resin</id>
```

Comment:
I think the `resin-3` plugin should be removed too

Reply:
done.